src/react-magic-script/magic-script-renderer.js

```diff
@@ -164,12 +164,7 @@ function shouldDeprioritizeSubtree(type, props) {
 //  type: string,
 //  props: Props
 function shouldSetTextContent(type, props) {
-  // Brian Vaughn:
-  //  Always returning false simplifies the createInstance() implementation,
-  //  But creates an additional child Fiber for raw text children.
-  //  No additional native views are created though.
-
-  return false;
+  return type === 'prism';
```

Comment:
what does shouldSetTextContent mean for a prism? (not pointing out anything wrong, just trying to understand what this does)

Reply:
If `shouldSetTextContent()` returns true, the next call would be to `createInstance()` for the current element and traversal would stop at this node (children of this element wont be traversed).